resources/views/layouts/navigation.blade.php

```diff
@@ -9,7 +9,11 @@
                         <x-application-logo class="block h-10 w-auto fill-current text-gray-600" />
                     </a>
                 </div>
-
+                <!--Search textbox -->
+                <form action="" method="GET">
```

Comment:
if you add the same classes you see on the navigation links div right below this form, it will disappear at smaller screen widths as we would expect. i.e. `class="hidden space-x-8 sm:-my-px sm:ml-10 sm:flex"`